### PR TITLE
[opentelemetry-operator] bump to 0.97.0

### PR DESCRIPTION
**Pull request by jaronoff97**
completes operator release for 0.97.0

### CODE REVIEW

**Review by Allex1 — APPROVED** [review]

File: charts/opentelemetry-operator/values.yaml [hunk 1]
```diff
@@ -160,7 +159,8 @@ manager:
 
   ## Container specific securityContext
   ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-container
-  securityContext: {}
+  securityContext:
+    {}
```
Comment (TylerHelmuth) [inline charts/opentelemetry-operator/values.yaml:163]: I assume this is a linter, but I'm not sure if it is equivalent to `securityContext: {}`.  Can we convert these back?

Comment (jaronoff97) [reply charts/opentelemetry-operator/values.yaml:163]: yeah this is a linter and is equivalent, i can swap it back...

Comment (jaronoff97) [reply charts/opentelemetry-operator/values.yaml:163]: i think its doing it because the comment is indented down. I'll fix that too

Comment (jaronoff97) [reply charts/opentelemetry-operator/values.yaml:163]: similar to this above:
```
    minAllowed: {}
    # cpu: 200m
    # memory: 100Mi
```

**Pull request merged**